config/webhook/manifests.yaml

```diff
@@ -135,3 +135,57 @@ webhooks:
           - UPDATE
         resources:
           - inferencegraphs
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: clusterservingruntime.serving.kserve.io
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: $(webhookServiceName)
+        namespace: $(kserveNamespace)
+        path: /validate-serving-kserve-io-v1alpha1-clusterservingruntime
+    failurePolicy: Fail
+    name: clusterservingruntime.kserve-webhook-server.validator
+    sideEffects: None
+    admissionReviewVersions: ["v1beta1"]
+    rules:
+      - apiGroups:
+          - serving.kserve.io
+        apiVersions:
+          - v1alpha1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - clusterservingruntimes
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: servingruntime.serving.kserve.io
```

Comment:
We will need to rename the validating webhook of the same name in ModelMesh

https://github.com/kserve/kserve/actions/runs/5741094894/job/15561730774#step:10:209

```
helm install kserve charts/kserve-resources/ --wait --timeout 180s

...

Error: INSTALLATION FAILED: 1 error occurred:
	* validatingwebhookconfigurations.admissionregistration.k8s.io "servingruntime.serving.kserve.io" already exists


make: *** [Makefile:128: deploy-helm] Error 1
Error: Process completed with exit code 2.
```

Reply:
#3063

Reply:
PR on the ModelMesh repo: https://github.com/kserve/modelmesh-serving/pull/415

Reply:
PR on the KServe repo: https://github.com/kserve/kserve/pull/3064

Reply:
@Jooho

Reply:
@sivanantha321 -- PR #3064 is merged now